README.md

```diff
@@ -47,36 +36,22 @@ commands from the main directory:
 ./mvnw -P coverage clean verify
 ```
 
-The Cloud Storage connector JAR can be found in `gcs/target/`. The BigQuery
-connector JAR can be found in `bigquery/target/`.
+The Cloud Storage connector JAR can be found in `gcs/target/` directory.
 
-## Adding the Cloud Storage and BigQuery connectors to your build
+## Adding the Cloud Storage to your build
```

Comment:
s/Adding the Cloud Storage to/Adding the Cloud Storage connector to/

Reply:
Good catch. Done!